card/lib/application_job.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationJob < ActiveJob::Base
+end
```

Comment:
I would have guessed that either (a) this is defined elsewhere and thus not needed or (b) it isn't and thus we can continue inheriting directly from ActiveJob::Base, but I'm probably missing something?